### DISCUSSION TITLE
IOS client app

### DISCUSSION THREAD

**Discussion by patrontheo** [Radiokot/photoprism-android-client#112] [General]
Hey @Radiokot,
Nice app you have done here!
I've been using Photoprism for a while, and just came across your app. I was wondering if you were planning to release an IOS version of your app ? It would be awesome. 
Would it require a lot of work to also support IOS ?
Thanks,
Theo

**Comment by Radiokot** [comment]
Hello Theo. 
I don't plan on doing anything with iOS, sorry.
